src/Servers/IIS/src/Microsoft.AspNetCore.Server.IIS/IISServerOptions.cs

```diff
@@ -1,10 +1,20 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Http;
```

Comment:
Redundant using?

Reply:
No, that's there fro the doc comment cref.